src/app/cli/src/graphql.ml

```diff
@@ -480,40 +481,47 @@ module Types = struct
                   account.Account.Poly.voting_for )
           ; field "stakingActive" ~typ:(non_null bool)
               ~doc:
-                "Actively staking. There is a lag between switching staking \
-                 keys and them appearing here as you may be in the middle of \
-                 a staking procedure with other keys."
+                "True if you are actively staking with this account - there \
```

Comment:
"Returns true if this account is actively staking - note that this may not yet have been updated if the staking key was changed recently"